Gems/Atom/RPI/Code/Include/Atom/RPI.Public/AssetTagBus.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Asset/AssetCommon.h>
+#include <AzCore/Name/Name.h>
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/std/containers/vector.h>
+#include <Atom/RPI.Public/AssetQuality.h>
+
+namespace AZ
+{
+    namespace RPI
+    {
```

Comment:
ditto here on the namespace lines - by consolidating you can remove one level of indent